pkg/plugin/state/node.go

```diff
@@ -77,7 +77,12 @@ type NodeResources[T constraints.Unsigned] struct {
 	// * Other pods are scheduled without going through our scheduler plugin
 	Reserved T
 
-	// Migrating is the amount of T that we expect will be removed by ongoing live migration.
+	// Committed is the sum of all Pods' <resource>.Reserved values, BEFORE applying any overcommit
+	// factors.
+	Committed T
```

Comment:
Why is it called `Commited`, what's the intuition?

Maybe something like `ReservedNominal` or `Effective` 🤔

Reply:
Reasoning for `Committed` was to match the naming for "overcommit", which allows "committing" more resources than actually exist on the node.

Reply:
Fair enough, makes sense.